### DISCUSSION TITLE
java客户端插入图片向量报错

### DISCUSSION THREAD

**Discussion by NotSky** [milvus-io/milvus#4604] [Q&A and General discussion] [answered]
[2021-01-12 16:33:41,252][ERROR][ENGINE][create][milvus_server] Failed to connect to database server: Too many connections。

**场景描述：将flink中的实时图片向量数据流，通过Milvus的java客户端0.9.1，将图片向量写入到Milvus，大概插到3万左右的时候报以上错误。**
-----------------------------------------------------
Milvus镜像版本：milvus:0.11.0-cpu-d101620-4c44c0
元数据管理：mysql5.7
Operating System: CentOS Linux 8 (Core)
CPE OS Name: cpe:/o:centos:centos:8
Kernel: Linux 4.18.0-147.5.1.el8_1.x86_64
Architecture: x86-64

![image](https://user-images.githubusercontent.com/5802285/104294968-d65e6600-54fa-11eb-9eeb-26f1e4e6b159.png)

**Comment by sahuang** [comment, marked as answer]
Can you also paste the error log (client/server side error) in order to locate the issue?

**Reply by sahuang** [reply]
A new issue here: https://github.com/milvus-io/milvus/issues/4605
